src/test/java/org/jenkinsci/plugins/kubernetes/cli/helpers/DummyCredentials.java

```diff
@@ -30,15 +30,15 @@ public static BaseStandardCredentials secretCredential(String credentialId) {
 
     public static BaseStandardCredentials certificateCredential(String credentialId) {
         byte[] storeFile = TestResourceLoader.loadAsByteArray("kubernetes.pkcs12");
-        CertificateCredentialsImpl.KeyStoreSource keyStoreSource = new CertificateCredentialsImpl.UploadedKeyStoreSource(
+        CertificateCredentialsImpl.KeyStoreSource keyStoreSource = new CertificateCredentialsImpl.UploadedKeyStoreSource(null,
                 SecretBytes.fromBytes(storeFile));
         return new CertificateCredentialsImpl(CredentialsScope.GLOBAL, credentialId, "sample", PASSPHRASE,
                 keyStoreSource);
     }
 
     public static BaseStandardCredentials brokenCertificateCredential(String credentialId) {
         byte[] storeFile = TestResourceLoader.loadAsByteArray("kubernetes.pkcs12");
-        CertificateCredentialsImpl.KeyStoreSource keyStoreSource = new CertificateCredentialsImpl.UploadedKeyStoreSource(
+        CertificateCredentialsImpl.KeyStoreSource keyStoreSource = new CertificateCredentialsImpl.UploadedKeyStoreSource(null,
```

Comment:
For myself: https://javadoc.jenkins.io/plugin/credentials/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl.UploadedKeyStoreSource.html#%3Cinit%3E(org.apache.commons.fileupload.FileItem,com.cloudbees.plugins.credentials.SecretBytes)